tasks/configure-jenkins-users.yml

```diff
@@ -2,8 +2,8 @@
 - name: Ensure Jenkins users exist
   community.general.jenkins_script:
     script: |
-      def userName = '{{ item.key }}'
-      def userPassword = '{{ item.value.password | default("") }}'
+      def userName = '{{ (item | dict2items)[0].key }}'
+      def userPassword = '{{ (item | dict2items)[0].value.password | default("") }}'
```

Comment:
Syntax-wise this is a step backward. Do we really have to replace `with_dict` by `loop`? Could we "process" `jenkins_users`  before passing it to `loop` so the syntax here is better?

Reply:
> 

I don't think so. We are basically "processing" it here. I think we might be able to improve things a bit by changing the data structure of `jenkins_users` from a list of dicts to a dict. In other words:

```yaml
jenkins_users:
  alice:
    password: "whatever"
  bob:
    password: "whatever"
```

This would of course require a minor version bump. I'll push a fixup to show what it would look like in practice.

Reply:
@ala-ableton What do you think about 0b7e621?